azurelinuxagent/common/protocol/extensions_goal_state_from_extensions_config.py

```diff
@@ -24,14 +24,14 @@
 from azurelinuxagent.common.exception import ExtensionsConfigError
 from azurelinuxagent.common.future import ustr
 from azurelinuxagent.common.protocol.extensions_goal_state import ExtensionsGoalState
-from azurelinuxagent.common.protocol.restapi import Extension, ExtHandler, ExtHandlerVersionUri, VMAgentManifest, \
-    VMAgentManifestUri, ExtensionState, ExtHandlerList, VMAgentManifestList, InVMGoalStateMetaData
+from azurelinuxagent.common.protocol.restapi import Extension, ExtHandler, VMAgentManifest, \
+    ExtensionState, ExtHandlerList, InVMGoalStateMetaData
 from azurelinuxagent.common.utils.textutil import parse_doc, parse_json, findall, find, findtext, getattrib, gettext, format_exception, \
     is_str_none_or_whitespace, is_str_empty
 
 
 class ExtensionsGoalStateFromExtensionsConfig(ExtensionsGoalState):
```

Comment:
can we test for agent_manifest from extension_config like we have for vm_settings.

Reply:
not sure i get your comment @nagworld9 ... the extension config stuff is existing code and has existing tests. is that what you meant?

Reply:
Unfortunately, the existing tests not covering agent manifests checks. I don't mind leaving it now but may be when you touch the test file again it's worthwhile to consider testing.
https://github.com/Azure/WALinuxAgent/blob/develop/tests/protocol/test_extensions_goal_state_from_extensions_config.py#L8

Reply:
yes, the existing code lacks some tests. retrofitting tests to that code would be costly. do feel free to add tests where you consider it worth the effore